configure.ac

```diff
@@ -503,6 +527,7 @@ case $host in
      AC_CHECK_LIB([shlwapi],      [main],, AC_MSG_ERROR(libshlwapi missing))
      AC_CHECK_LIB([iphlpapi],      [main],, AC_MSG_ERROR(libiphlpapi missing))
      AC_CHECK_LIB([crypt32],      [main],, AC_MSG_ERROR(libcrypt32 missing))
+     AC_CHECK_LIB([userenv],      [main],, AC_MSG_ERROR(libuserenv missing))
```

Comment:
Do we need `libuserenv` for anything other than Rust?

Reply:
What is libuserenv?

Reply:
Good point, this should be gated.
It may not even be necessary now that we've slimmed down.

Reply:
mingw+rust builds fine without this now, so I guess we can just remove it.